### PR TITLE
add astsbins

### LINKED ISSUE #23: add Jack L
#### How to get started

1) star this repository (top right) so that we can help you get more visibility over your open source contributions
2) follow the [README.md instructions](https://github.com/the-fast-tracked-programmer/first-contributions) to learn how to make your first contribution to an open source project

#### If you get stuck and need some guidance or support

Jump into our discord channel [#first-contributions](https://discord.com/invite/2QaYgquy6R) and request some help from an experienced mentor

### PR DESCRIPTION
**Pull request by astsbins**
closes #23

### CODE REVIEW

File: bharat-patodi.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-## Hi, I'm Bharat Patodi
-Glad to be here.
-Hope to have a great time working with open source.
-Thanks Adam and Adarsh.
```
Comment (Subskii) [inline bharat-patodi.md:4]: hey @astsbins wondering if you intended to change this file?

File: astbins.md [hunk 2]
```diff
@@ -0,0 +1 @@
+## Hello, I'm Jack
```
Comment (Subskii) [inline astbins.md:1]: just need to move this one into the `first-contributions` folder within the repo e.g. `first-contributions/astbins.md`

File: bharat-patodi.md (hunk 1 above)
Comment (Subskii) [inline bharat-patodi.md:4]: wondering if you intended to change this file?

**Review by Subskii — CHANGES_REQUESTED** [review]
Can re-request a review once you've addressed those 2 issues :smiley:

File: bharat-patodi.md (hunk 1 above)
Comment (astsbins) [reply bharat-patodi.md:4]: Yeah, it was intentional, but now I realise that its a mistake. There is supposed to be one file per member.

**Comment by astsbins** [comment]
Is it okay now?

File: astbins.md (hunk 2 above)
Comment (Subskii) [reply astbins.md:1]:
```
first-contributions$ git mv astbins.md first-contributions/astbins.md
first-contributions$ git commit -m "move astbins.md to first-contributions/astbins.md"
```

**Comment by Subskii** [comment]
Just need to move your file into the `first-contributions/` folder.

Instructions could be clearer we'll get this updated soon.

**Comment by Subskii** [comment]
Once you've done that and pushed up your changes, re-request a review and we'll get this thing merged for you!

**Comment by Subskii** [comment]
Oh and finally make sure you update the pull request description with `closes #65` so the issue is properly linked

**Comment by astsbins** [comment]
closes #65

**Comment by astsbins** [comment]
is the pull request description just the comments here?

**Comment by Subskii** [comment]
> is the pull request description just the comments here?

Doesn't seem to link the pull request to the issue if you reference it in the comments :(

Check out [this comment](https://github.com/the-fast-tracked-programmer/first-contributions/pull/73#issuecomment-913349659) for more details

**Comment by astsbins** [comment]
closes #23

**Comment by astsbins** [comment]
Hopefully i got it right this time XD

**Review by Subskii — APPROVED** [review]
BINGO!

Great work @astsbins :rocket:!

**Pull request merged**